### ISSUE TITLE
565507EF - Spider-Man 4 (Unreleased)

### ISSUE THREAD

**Issue by icup321** [xenia-project/game-compatibility#1825]
Tested on https://github.com/xenia-project/xenia/commit/44847abb98bf39b982bda2aa185f9f65ff3b1281

# Issues:
* Game renders 3D environments very badly by default because some depth buffer precision is lost during a round-trip between the X360 and the PC depth formats, so a fix is necessary in order to at least see anything. To fix this, set depth_float24_conversion to "truncate" in Documents\Xenia\xenia.config.toml (however, it's not recommended for games without this issue, as it impacts GPU performance a bit). Or, you can switch to more accurate, but in many cases slower, software color packing and depth testing if your graphics card supports the Rasterizer-Ordered View functionality required for that — see "Render target path" at: https://github.com/xenia-project/xenia/wiki/Options#Render_target_path

* Aside from that, expect other Xenia-specific issues like black bands on the ground, crashes, falling through the ground, low framerate, etc. Many of these issues are the result of Xenia not being able to emulate games using the Titanium engine properly.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/7144552/xenia.zip)

# Screenshot(s):
![](https://media.discordapp.net/attachments/703633675480334443/885886163167100978/spiderman4xeniaworking.png)

# Labels:
cpu-jit-problems
cpu-unimplemented-instruction
gpu-drawing-corrupt
state-gameplay
tech-engine-titanium

**Comment by Triang3l** [comment]
Thanks for the report! I haven't added `cpu-unimplemented-instruction` though — did it crash with a report of some unimplemented instruction in the log and may be a good test case for it, or are you referring to the overall incorrect CPU behavior?

**Comment by icup321** [comment]
Yes, I was copying it from Prototype's page since that seems to include it as well, was debating whether I should include it or not, but I put it in just in case.

**Comment by PaHaNchickT** [comment]
Hi! Where this release could be downloaded? Or how could I contact with you?

**Comment by Psychological536** [comment]
how the fuck do i run this bullshit i go to files then click on it in my library and everything close what the fuck???????????????????????????????????????????

**Comment by Psychological536** [comment]
how do i run this everytime i click on the file on xenia it close wtf?????

**Comment by Margen67** [comment]
@Psychological536 Tech support belongs on the [Discord server](https://discord.gg/Q9mxZf9).

**Comment by Sodicious** [comment]
Heads up: if you dont have "depth_float24_conversion" in your config, **set "depth_float24_convert_in_pixel_shader"** and **"depth_float24_round"** to **true**. It worked for me, and seeing as there is no mention to this here, why not put a mention to it?
